doc/source/getting_started/WorkingWithInferenceData.ipynb

```diff
@@ -13,11 +13,13 @@
   },
```

Comment:
<p>This is rST syntax for directives, in notebooks myst has to be use: https://myst-parser.readthedocs.io/en/latest/using/syntax.html#syntax-directives. In this case it should be:</p><p> </p><pre class="ql-syntax" spellcheck="false"><code></code>`{note}
text of note
<code></code>`
</pre><p> </p><p>I can' seem to get this to format properly though</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>